app/src/admin/AndroidManifest.xml

```diff
@@ -20,9 +20,11 @@
         xmlns:tools="http://schemas.android.com/tools"
         android:allowBackup="true"
         android:label="@string/app_name"
+        android:requestLegacyExternalStorage="true"
```

Comment:
This is not going to work on android 11+.  Instead of trying to access the storage use intent to open file manager on android 11+.